Gruntfile.js

```diff
@@ -362,6 +362,18 @@ export default Vex;`;
     ]
   );
 
+  // `grunt watchDevelop`
+  // This is the fastest way to build vexflow-debug-with-tests.js.
+  // Open tests/flow.html to see the test output.
+  grunt.registerTask(
+    'watchDevelop',
```

Comment:
The camel case here keeps throwing me off. Why don't we use `watch:develop`, `watch:production`, etc.

(Also, let's use `dev|prod` or `development|production` to keep things consistent. I prefer the short form, but not a strong opinion.)

Reply:
Thanks for the feedback. I had to improve the build process for the ESM branch, so some of this is already outdated, ha. 

We can revisit this topic when I submit my ESM PR for review, and you can look over the new Gruntfile and let me know what you'd like changed.

Are you against camelCase in in the Gruntfile in general?

I'm not entirely sure a colon works for registerTask(). I've seen it used to separate `task:target` during the config: https://gruntjs.com/configuring-tasks  I've also seen it referenced as a way to pass command line arguments to a task: https://gruntjs.com/creating-tasks#basic-tasks

Anyways, I can investigate to see if/how the colon form can be used here.

Reply:
Not against camelCase in general in the Gruntfile -- mostly for things run in the commandline, where lower-case or lower_case is more common. So I prefer to run `grunt watch-dev` than `grunt watchDev`. Open to even `grunt dev`.